services/src/handlers/workflows.rs

```diff
@@ -348,13 +353,13 @@ async fn resolve_provenance<C: Context>(
 }
 
 /// parameter for the dataset from workflow handler (body)
-#[derive(Clone, Debug, Deserialize, Serialize, Component)]
-#[component(example = json!({"name": "foo", "description": null, "query": {"spatialBounds": {"upperLeftCoordinate": {"x": -10.0, "y": 80.0}, "lowerRightCoordinate": {"x": 50.0, "y": 20.0}}, "timeInterval": {"start": 1_388_534_400_000_i64, "end": 1_388_534_401_000_i64}, "spatialResolution": {"x": 0.1, "y": 0.1}}}))]
+#[derive(Clone, Debug, Deserialize, Serialize, ToSchema)]
```

Comment:
Just for curiosity: What is the difference?

Reply:
it's just renamed in utoipa to more closely match the OpenAPI specification on what a component and what a schema is.